support-payment-api/src/main/scala/model/AcquisitionData.scala

```diff
@@ -13,8 +12,8 @@ case class AcquisitionData(
   referrerUrl: Option[String],
   componentId: Option[String],
   campaignCodes: Option[Set[String]],
-  componentType: Option[ComponentType],
-  source: Option[AcquisitionSource],
```

Comment:
consistent with the scala acquisition model, these are now just strings

Reply:
in general I'm not a massive fan of strings everywhere, because they're so easy to mix up and therefore in this case mess up the analytics, but consistency trumps that and I think you added more tests in the last PR which should help.

Reply:
That model was introduced for BigQuery integration and has now replaced the thrift model. It wouldn't be hard to re-introduce e.g. a `ComponentType` type if we want to. Now that we've escaped thrift it's much easier to add new values to enums